api_factory/templates/$namespace/$name_$version/$service/client.py.j2

```diff
@@ -1,23 +1,17 @@
 {% extends '_base.py.j2' %}
 
 {% block content %}
-import functools
 import pkg_resources
 from typing import Mapping, Optional, Sequence, Tuple, Union
-{%- if service.has_field_headers %}
-from urllib import parse
-{%- endif %}
 
 from google.api_core import gapic_v1
-{%- if service.has_lro %}
-from google.api_core import operation
-{%- endif %}
 from google.api_core import retry
 from google.auth import credentials
 
-{% for package, pb2_module in service.pb2_modules -%}
-from {{ package }} import {{ pb2_module }}
+{% for package, python_module in service.python_modules -%}
+from {{ package }} import {{ python_module }}
 {% endfor %}
+from ..utils import dispatch
```

Comment:
Any plans on moving this to api_core?

Reply:
Yes. However, since `api_core` is GA, I want to let things go through some testing before shifting them there. Ultimately, I do not want a `utils.py` to be generated, however.

Reply:
Ack.